models/quickbooks__balance_sheet.sql

```diff
@@ -8,6 +8,8 @@ with general_ledger_by_period as (
 final as (
     select
         period_first_day as calendar_date,
+        period_first_day,
```

Comment:
maybe we should note here as well in a comment that calendar_date will be deprecated

Reply:
good point! added